.github/workflows/background-blur-integration.yml

```diff
@@ -0,0 +1,79 @@
+  name: Background Blur Integration Workflow
```

Comment:
Do we need a whole workflow for background-blur? Can we nest it under the video workflow?